src/Devices/PHControl.h

```diff
@@ -5,6 +5,7 @@ class PHControl {
   // Class variable
   static PHControl* _instance;
   // instance variables
+  unsigned long lastSwitchMS = 0;
```

Comment:
This seems out of place since you are only changing temperature.

Remove this line and include it in the ph change.